src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -578,6 +618,9 @@
     </Touch>
   </Target>
 
+  <Target Name="ExtractToolPackage"
+          DependsOnTargets="RepoBuild;$(ExtractToolPackageDependsOn);ExtractToolPackageCore" />
```

Comment:
Can we now remove the separate `ExtractToolPackageCore` target that isn't necessary anymore?

Reply:
I'll try and see what happens. I've noticed that sometimes, MSBuild doesn't like running target dependencies when a target's condition is unmet. Still haven't figured out the exact scenarios where that happens though.

Reply:
Yeah based on other similar patterns in the repo, I'm going to keep the "Core" target separate.